Fiber2D/ActionSequence.swift

```diff
@@ -35,7 +35,9 @@ public struct ActionSequenceContainer: ActionContainer, Continous {
         if found == 1 {
             if last == -1 {
                 // action[0] was skipped, execute it.
-                actions[0].start(with: target)
+                if let target = target {
```

Comment:
this should be guarded in the beginning of the method, otherwise it is a wrong logic